SketchUp/length.rb

```diff
@@ -42,15 +42,15 @@ class Length < Float
   #   length1 = 12.to_l
   #   length2 = 11.to_l
   #   if length1 < length2
-  #     puts "length1 is less than length2"
+  #     puts("length1 is less than length2")
```

Comment:
By convention `puts` and `p` doesn't use parentheses. They are exceptions to the rule.